Run from source

## Griddle version
source 2017-12-17, 1e6346a Merge pull request #771 from dahlbyk/gh672
## Expected Behavior
Story book should work
## Actual Behavior
ERROR in ./stories/index.tsx
(923,31): error TS2339: Property 'searchString' does not exist on type '{}'.

ERROR in ./stories/index.tsx
(947,22): error TS2339: Property 'searchString' does not exist on type '{}'.

ERROR in ./stories/index.tsx
(947,43): error TS2339: Property 'data' does not exist on type '{}'.

ERROR in ./stories/index.tsx
(948,17): error TS2339: Property 'data' does not exist on type '{}'.

ERROR in ./stories/index.tsx
(949,76): error TS2339: Property 'searchString' does not exist on type '{}'.
## Steps to reproduce
`git clone https://github.com/GriddleGriddle/Griddle.git && cd Griddle && npm install && npm run storybook`
I use node v8.9.3

> > start-storybook -p 6006
> 
> @kadira/storybook v2.35.3
> 
> => Loading custom .babelrc
> => Loading custom webpack config.
> 
> React Storybook started on => http://localhost:6006/
> 
>  We will collect some anonymous usage stats of how you use storybook.
>  See why?: https://getstorybook.io/tracking
> 
> ts-loader: Using typescript@2.5.3 and /home/user/workspace/Griddle/tsconfig.json
> webpack built 46757a42dcf954be8b77 in 8341ms
> Hash: 46757a42dcf954be8b77
> Version: webpack 1.15.0
> 

Are there any additional requirements to run from source?

--- a/package.json
+++ b/package.json
@@ -32,7 +32,7 @@
   "devDependencies": {
     "@kadira/storybook": "^2.5.2",
     "@types/react": "^15.0.34",
-    "@types/react-redux": "^5.0.3",
+    "@types/react-redux": "^5.0.14",
     "ava": "^0.17.0",
     "babel-cli": "^6.23.0",
     "babel-core": "^6.0.0",

--- a/stories/index.tsx
+++ b/stories/index.tsx
@@ -73,7 +73,13 @@ const EnhanceWithRowData = connect((state, props) => ({
 
 const EnhancedCustomComponent = EnhanceWithRowData(MakeBlueComponent);
 
-function testReducer(state = { count: 1}, action) {
+interface TestState {
+  count: number;
+  data?: any;
+  searchString?: string;
+}
+
+function testReducer(state: TestState = { count: 1 }, action: any) {
   switch (action.type) {
     case 'INCREMENT':
       return { ...state, count: state.count + 1};
@@ -919,7 +925,7 @@ storiesOf('Griddle main', module)
 
     const setSearchStringActionCreator = searchString => ({ type: 'SET_SEARCH_STRING', searchString })
     const CustomFilterConnectedComponent = reduxConnect(
-      state => ({
+      (state: TestState) => ({
           searchString: state.searchString,
       }),
       dispatch => ({
@@ -943,7 +949,7 @@ storiesOf('Griddle main', module)
     );
 
     const SomePageConnected = reduxConnect(
-      state => ({
+      (state: TestState) => ({
         data: !state.searchString ? state.data :
           state.data.filter(r =>
             Object.keys(r).some(k => r[k] && r[k].toString().indexOf(state.searchString) > -1)),

--- a/stories/index.tsx
+++ b/stories/index.tsx
@@ -918,7 +918,7 @@ storiesOf('Griddle main', module)
 
     const CustomFilterComponent = (props) => (
       <input
-        value={props.searchString}
+        value={props.searchString || ''}
         onChange={(e) => { props.setSearchString(e.target.value); }}
       />
     );

